tests/tests_lite/helpers/utils.py

```diff
@@ -29,3 +33,25 @@ def set_random_main_port():
     reset_seed()
     port = RANDOM_PORTS.pop()
     os.environ["MASTER_PORT"] = str(port)
+
+
+@contextmanager
+def no_warning_call(expected_warning: Type[Warning] = UserWarning, match: Optional[str] = None):
```

Comment:
@carmocca I'm not bothering copying from tests pytorch right now, but perhaps this could move to lightning-utilities.

Reply:
Thanks for flagging. I'll move it.